### DISCUSSION TITLE
How to make RabbitMQ MQTT plugin use port 1884

### DISCUSSION THREAD

**Discussion by sagarganachari** [rabbitmq/rabbitmq-server#2875] [Questions] [answered]
Dear Experts ,

I installed RabbitMQ in linux server in that i installed MQTT plugin to communicate with MQTT CLients, I WANT TO CHANGE PORT NUMBER DEFAULT 1883 TO 1884 ,How to change it...

i am getting error after i enable plug in with port1883

![image](https://user-images.githubusercontent.com/76048188/110241518-b8493b00-7f6a-11eb-91ec-eef7661d58e7.png)

**Comment by michaelklishin** [comment, marked as answer]
As described in the [TCP listeners section in the MQTT guide](https://www.rabbitmq.com/mqtt.html#tcp-listeners).

**Comment by michaelklishin** [comment]
The exception you are looking at has a hint: `eaddrinuse`. The port is already used by another MQTT broker.
